finality-aleph/src/network/manager/discovery.rs

```diff
@@ -84,7 +84,7 @@ impl<M: Multiaddress> Discovery<M> {
 
         let missing_authorities = handler.missing_nodes();
         let node_count = handler.node_count();
-        debug!(target: "aleph-network", "{:?}/{:?} authorities known for session {:?}.", node_count.0-missing_authorities.len(), node_count, handler.session_id());
+        info!(target: "aleph-network", "{}/{} authorities known for session {}.", node_count.0-missing_authorities.len(), node_count.0, handler.session_id().0);
```

Comment:
Sometimes it's useful to know the keys of the authorities as well.

Reply:
Yes -- more info about current session authorities is logged at a different place actually. This line prints only some minimal info on the status of network connectivity between authorities :)